.github/workflows/idol-notifications.yml

```diff
@@ -48,11 +52,38 @@ jobs:
           restore-keys: yarn-
       - name: Install
         run: yarn
-      - name: Profile update notifications
+      - name: TEC request notification
         env:
           FIREBASE_PRIVATE_KEY: ${{ secrets.FIREBASE_PRIVATE_KEY }}
           FIREBASE_PRIVATE_KEY_ID: ${{ secrets.FIREBASE_PRIVATE_KEY_ID }}
           OAUTH_CLIENTID: ${{ secrets.OAUTH_CLIENTID}}
           OAUTH_CLIENT_SECRET: ${{ secrets.OAUTH_CLIENT_SECRET }}
           OAUTH_REFRESH_TOKEN: ${{ secrets.OAUTH_REFRESH_TOKEN }}
         run: yarn workspace backend send-tec-notifications
+
+member-archive-reminder:
+  if: contains(github.event.schedule, '0 14 20 1,8 *')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
In Oscar's PR #607, he updated the workflows, so this should be changed to `actions/checkout@v4`, along with lines 71 and 75. :)